ultimaker_pc_black.xml.fdm_material

```diff
@@ -48,17 +48,5 @@ Generic PC profile. The data in this file may not be correct for your specific m
             </hotend>
         </machine>
 
-        <machine>
-           <machine_identifier manufacturer="Cartesio bv" product="cartesio" />
-           <setting key="hardware compatible">yes</setting>
-           <setting key="print temperature">245</setting>
-           <setting key="heated bed temperature">135</setting>
-           <setting key="standby temperature">160</setting>
-           <setting key="print cooling">0.0</setting>
-           <hotend id="0.25 mm" />
-           <hotend id="0.4 mm" />
-           <hotend id="0.8 mm" />
-       </machine>
```

Comment:
Are you sure you wish to remove support for Ultimaker PC Black material, but retain support for generic PC?